api/models.py

```diff
@@ -30,14 +25,13 @@ class Collection(models.Model):
     start_datetime = models.DateTimeField(null=True, blank=True)
     end_datetime = models.DateTimeField(null=True, blank=True)
 
-    def save(self, *args, **kwargs):
```

Comment:
also why get rid of this?

Reply:
I think this should be a merge conflict? This is new code you wrote that I never had...

Reply:
wait no I did have this. I may have deleted this because its broken (the *args **kwargs thing)

Reply:
definitely re-add then 👌

Reply:
this should be good

Reply:
:+1: